js/js-array-methods/js-array-methods_for-each/js/index.js

```diff
@@ -19,4 +19,10 @@ const colors = [
   "#A5A5A5",
 ];
 
-colors.forEach();
+colors.forEach((color) => {
+  const div = document.createElement("div");
+  div.classList.add("color-box");
+  div.style.backgroundColor = "" + color + "";
```

Comment:
the double quotes "" are not necessary, if you are just referring to the 'color' variable.